install/kots/manifests/kots-links.yaml

```
@@ -0,0 +1,24 @@
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: gitpod
+  labels:
+    app.kubernetes.io/name: gitpod
+    app.kubernetes.io/version: "1.0.0"
+spec:
+  selector:
+    matchLabels:
+     app.kubernetes.io/name: gitpod
+  descriptor:
+    links:
+      - description: Gitpod
+        url: https://www.gitpod.io
+      - description: Gitpod Docs
+        url: https://www.gitpod.io/docs
+      - description: Gitpod Source Code
+        url: https://www.github.com/gitpod-io/gitpod
+      - description: My Gitpod
```

```suggestion
      - description: Your Gitpod Dashboard
```
Might be worth moving this to the very top as well.

---

To be honest, I haven't known what “My Gitpod” resp. “Your Gitpod Dashboard” actually means until I looked at the URL. What about using the domain in the description like `Open https://{{repl ConfigOption "domain" }}` instead?